tor_core/__init__.py

```diff
@@ -0,0 +1,32 @@
+import logging
+import sys
+import time
+
+import prawcore
+
+from tor_core.config import config
+from tor_core.helpers import explode_gracefully
+
+__version__ = '3.0.0'
+
+
+default_exceptions = (prawcore.exceptions.RequestException)
+
+def run_until_dead(func, bot_name=None, exceptions=default_exceptions):
```

Comment:
While I agree that this method should exist, I'm not sure I would agree this file is the best place for it. I usually think of __init__.py as a metadata-only kind of file. Is there a reason this seemed appropriate as-is?

Perhaps it should be placed in a file dedicated to bot helpers?